lib/controller.py

```diff
@@ -521,3 +523,97 @@ def do_dun(self, line):
     def do_undun(self, line):
         results = self.cluster.infoUndun(self.mods['line'], nodes=self.nodes)
         self.view.dun(results, self.cluster, **self.mods)
+
+@CommandHelp('"collectinfo" is used to collect system stats on the local node.')
+class CollectinfoController(CommandController):
+    def collect_local_file(self,src,dest_dir):
+        shutil.copy2(src, dest_dir)
+        
+    def write_log(self,collectedinfo,src_file='/var/log/aerospike/*log'):
+        aslogdir = '/tmp/as_log_' + str(time.time())
```

Comment:
>>> aslogdir
'as_log_1425024563.54'
>>> tempfile.mkdtemp('',aslogdir)
'/tmp/as_log_1425024563.54_iu1Uq

The suffix makes it non-standard to be used.